udata/core/dataset/forms.py

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from udata.auth import current_user
-from udata.forms import Form, ModelForm, UserModelFormMixin, fields, validators, widgets
+# from udata.auth import current_user
```

Comment:
I don't think commenting dead code is a good practice when we use a CVS, let's drop it.